### PR TITLE
[UI Design] (Office UI Fabric React) Updated instructions to reflect recent changes to Yo Office.

### LINKED ISSUE #1161: info in 'Get started with Fabric React' is outdated
Info in this article is outdated (because Yo Office has changed significantly since the article was last updated):  https://docs.microsoft.com/en-us/office/dev/add-ins/design/using-office-ui-fabric-react.

### PR DESCRIPTION
**Pull request by kbrandl**
Fixes #1161 

Since significant changes have been made, I'd recommend reviewing the revised article on the staging site (https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/using-office-ui-fabric-react?branch=kb-fix-1161) instead of (or in addition to) just reviewing diff's here in this PR.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c8f4e58](https://github.com/OfficeDev/office-js-docs-pr/pull/1165/commits/c8f4e585fe92c2ce2f4c5f7d43389b37e086023e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-with-react-component.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-with-react-component.png) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-yo-default.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-yo-default.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-react.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/yo-office-word-react.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907102206433514-1165%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DAFq4ghCsyEwe3B9zj4n2h%2FWp2gpR5I32VrIwyDpvGw%3D&st=2019-07-10T22%3A03%3A03Z&se=2019-08-10T22%3A08%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/design/using-office-ui-fabric-react.md [hunk 1]
```diff
@@ -1,56 +1,109 @@
...
+    ```
+
+    - **Choose a project type:** `Office Add-in Task Pane project using React framework`
+    - **Choose a script type:** `TypeScript`
+    - **What do you want to name your add-in?** `My Office Add-in`
+    - **Which Office client application would you like to support?** `Word`
+
+    ![Yeoman generator](../images/yo-office-word-react.png)
+
+    After you complete the wizard, the generator will create the project and install supporting Node components.
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/design/using-office-ui-fabric-react.md]: FYI, I'll also make this change everywhere else this statement appears in the docs.

File: docs/design/using-office-ui-fabric-react.md [hunk 2]
```diff
@@ -1,56 +1,109 @@
...
+
+2. Navigate to the root folder of the project.
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/design/using-office-ui-fabric-react.md]: Good feedback; I'll also make this change to all of the quick starts (where the articles are similarly structured).

File: docs/design/using-office-ui-fabric-react.md [hunk 3]
```diff
@@ -1,56 +1,109 @@
...
+    cd "My Office Add-in"
+    ```
+
+### Try it out
+
+1. Complete the following steps to start the local web server and sideload your add-in.
 
-## Step 1 - Create your project with the Yeoman generator for Office
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides.
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/design/using-office-ui-fabric-react.md]: Hmmmm. This behavior isn't specific only to add-ins created using Yo Office -- all Office Add-ins (regardless of how they're created) should use HTTPS -- so, the first sentence is intended to convey that. It just so happens that Yo Office gives you a cert that you can use while developing/testing (as the second sentence explains).  I'm going to leave this one as-is for now, and we can revisit later if need be.

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the review and feedback! I've made some updates; please have another look when you have the chance. Thanks!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c2512e4](https://github.com/OfficeDev/office-js-docs-pr/pull/1165/commits/c2512e4119af0f4dd5624d04ad3feda44a87db64)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-with-react-component.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-with-react-component.png) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-yo-default.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-yo-default.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-react.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/yo-office-word-react.png) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-angular.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-jquery.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-react.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907112037445097-1165%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EJVFqOQ2wPWnYgMHOLGSjAuEKJHNMTnsOrEmuKzTB3U%3D&st=2019-07-11T20%3A33%3A47Z&se=2019-08-11T20%3A38%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d9dcc94](https://github.com/OfficeDev/office-js-docs-pr/pull/1165/commits/d9dcc94b4bc87c061b5ede521122fc2f512a685a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-with-react-component.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-with-react-component.png) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-yo-default.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-yo-default.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-react.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/yo-office-word-react.png) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-angular.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-jquery.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-react.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907112041260777-1165%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kYAv8f1nPMpzLRW4KMx4gobv2ykvCvF1nnThTAd8n80%3D&st=2019-07-11T20%3A37%3A38Z&se=2019-08-11T20%3A42%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0e40975](https://github.com/OfficeDev/office-js-docs-pr/pull/1165/commits/0e40975520e95c868d65b1f1b593b421df4a3473)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-with-react-component.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-with-react-component.png) | :white_check_mark:Succeeded | |
[docs/images/word-task-pane-yo-default.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/word-task-pane-yo-default.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-react.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/images/yo-office-word-react.png) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server-excel.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/includes/quickstart-yo-start-server-excel.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-angular.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-jquery.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-quickstart-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/excel-quickstart-react.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/onenote-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/onenote-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/powerpoint-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/powerpoint-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/project-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/project-quickstart.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/word-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-1161/docs/quickstarts/word-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907112048406688-1165%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aXiVoMkzQx1E0tLvlVcUATit3AHOV3EzC16vEbfRne0%3D&st=2019-07-11T20%3A44%3A48Z&se=2019-08-11T20%3A49%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**